### PR TITLE
Multiple file log dialog

### PR DESCRIPTION
**Pull request by donob4n**
Not very fashion but it saves a lot of clicks if you want to see many log files from one (or more) qubes.

logdialog

Probably a list should be a better solution than buttons.

### CODE REVIEW

File: qubesmanager/log_dialog.py [hunk 1]
```diff
@@ -52,8 +51,32 @@ def __init__(self, app, log_path, parent=None):
         self.__init_log_text__()
 
     def __init_log_text__(self):
+        butts_in_row = 3
```
Comment (marmarek) [inline qubesmanager/log_dialog.py:54]: :smile: 
```suggestion
        btns_in_row = 3
```

**Review by marmarek — CHANGES_REQUESTED** [review]

File: qubesmanager/log_dialog.py (hunk 1 above)
Comment (donob4n) [reply qubesmanager/log_dialog.py]: lol

Comment (marmarek) [reply qubesmanager/log_dialog.py]: But you need to update usages of the variable too.

Comment (donob4n) [reply qubesmanager/log_dialog.py]: Yes sorry, it seems that there are also conflicts with other PR. I will check tonight.

**Comment by marmarek** [comment]
If there are no logs, the logs window is still opened after a message.

**Comment by donob4n** [comment]
Yes I noticed  it when I added the warning dialog. I tried to close the window using accept(), close(), reject(),... but all failed probably because they were called from _init__(). Maybe the best solution is moving the file checking to qube_manager.py and show the error from there.

**Comment by marmarek** [comment]
On Thu, Dec 10, 2020 at 03:06:32PM -0800, donob4n wrote:
> Yes I noticed  it when I added the warning dialog. I tried to close the window using accept(), close(), reject(),... but all failed probably because they were called from _init__(). Maybe the best solution is moving the file checking to qube_manager.py and show the error from there.

Yes.

**Review by marmarek — APPROVED** [review]

**Pull request merged**